bigquery/README.md

```diff
@@ -2,50 +2,50 @@
-  ```
-* Change project ids in `dataset_config/platinum_genomes/facet_fields.csv`
-* If `~/.config/gcloud/application_default_credentials.json` doesn't exist, create it by running `gcloud auth application-default login`
-* `docker-compose up --build`
-* View Elasticsearch index:
- `http://localhost:9200/platinum_genomes/_search?pretty=true`
+* If `~/.config/gcloud/application_default_credentials.json` doesn't exist,
+create it by running `gcloud auth application-default login`.
+* Determine the project that will be billed for querying the BigQuery tables.
+Minimally, your account must have `bigquery.jobs.create` permissions on this
+project.
+* Run the indexer:
+  `BILLING_PROJECT_ID=<billing_project_id> docker-compose up --build`
+* View the Elasticsearch index at
+ `http://localhost:9200/platinum_genomes/_search?pretty=true`.
 
 If you want to run the Data Explorer UI on this dataset, follow the instructions
-below. Note that you will have to reindex the data into an Elasticsearch
+below. Note that you will have to re-index the data into an Elasticsearch
 container from the [data-explorer repo](https://github.com/DataBiosphere/data-explorer/).
 
 ### Index a custom dataset locally
 
-* If `~/.config/gcloud/application_default_credentials.json` doesn't exist, create it by running `gcloud auth application-default login`
+* If `~/.config/gcloud/application_default_credentials.json` doesn't exist,
+create it by running `gcloud auth application-default login`.
 * Setup config files.
   * Create `dataset_config/<my_dataset>`. Copy `dataset_config/template/*` to this directory.
   * Edit config files; instructions are in the files. Read
   [Overview](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#overview)
   for some background information.
-* Run Elasticsearch.
-  * If you intend to run the Data Explorer UI from the [data-explorer repo](https://github.com/DataBiosphere/data-explorer/)
-    after this, run inside the data-explorer repo:
+* Run Elasticsearch:
+  * If you intend to run the [Data Explorer UI](https://github.com/DataBiosphere/data-explorer/)
+  after this, run inside the data-explorer repo:
     ```
     docker-compose up -d elasticsearch
     ```
   * If you do not intend to run the Data Explorer UI after this, and just want
-    to inspect the index in Elasticsearch, run inside this repo:
+  to inspect the index in Elasticsearch, run inside this repo:
     ```
     docker-compose up -d elasticsearch
     ```
+* Determine the project that will be billed for querying the BigQuery tables.
+Minimally, your account must have `bigquery.jobs.create` permissions on this
+project.
 * Run the indexer:
-`DATASET_CONFIG_DIR=dataset_config/<my dataset> docker-compose up --build indexer`
+  ```
+  BILLING_PROJECT_ID=<billing_project_id> DATASET_CONFIG_DIR=dataset_config/MY_DATASET docker-compose up --build indexer
```

Comment:
I would like to keep `<my dataset>`.

I think with `DATASET_CONFIG_DIR=dataset_config/<my dataset>`, it's more clear what you should change, than `DATASET_CONFIG_DIR=dataset_config/MY_DATASET`.

Reply:
Done